tests/functional/spec/features/submitted_response/submitted_response.js

```diff
@@ -0,0 +1,22 @@
+import NameBlockPage from "../../../generated_pages/view_submitted_response/name.page.js";
```

Comment:
The card said no function tests, however although this will need a tweak when we have a link, I saw no reason why a light weight test couldn't be put in